aws/table_aws_cloudtrail_event_data_store.go

```diff
@@ -0,0 +1,184 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudtrailEventDataStore(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudtrail_event_data_store",
+		Description: "AWS CloudTrail Event Data Store",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AnyColumn([]string{"event_data_store_arn"}),
+			Hydrate:    getCloudTrailEventDataStore,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCloudTrailEventDataStores,
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the event data store.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "event_data_store_arn",
```

Comment:
```suggestion
				Name:        "arn",
```

I think we usually normalize the `arn` columns to just be `arn` (so it's easy to find and so our mod controls work well with them).